docs/ref/middleware.txt

```diff
@@ -155,6 +155,9 @@ It will NOT compress content if any of the following are true:
 * The request (the browser) hasn't sent an ``Accept-Encoding`` header
   containing ``gzip``.
 
+If the response has an ``ETag`` header, the header will be changed to a weak
+``ETag`` to comply with :rfc:`7232`.
```

Comment:
Could we link to a specific section of the RFC? Let me know and I'll make the change when merging.